### ISSUE TITLE
Facebook share button on mobile does not open the app

### ISSUE THREAD

**Issue by Jhonan321** [nygardk/react-share#208]
Hello, first of all, thank you for your work making us easier to share content with the component, I have been trying only with the demo link from my cellphone and the only one button that doesn't work opening the native app is the Facebook share button, with Twitter, LinkedIn and Email share button worked (the only ones we need right now), but with Facebook button did not, I saw in my android configuration and asked to test this from a couple of friends cell phones, and they told me the same, it is opening in a new tab in chrome mobile, and not in the facebook app, is there a way to solve this? 

Thank you in advance

**Comment by IshaanKanwar** [comment]
Yes im facing the same problem. Its opening a new browser window instead of the facebook application

**Comment by jadus** [comment]
Hi, any news about this ?
Or alternative ?

Thanks a lot

**Comment by huepf** [comment]
This requires deep links instead of "normal" links. Would love to see this functionality.

**Comment by vnnycnza** [comment]
It's opening a new browser window with an option "Share with your Facebook App"

**Comment by Julian-Jurai** [comment]
Any updates?

**Comment by ducdev** [comment]
Any updates?

**Comment by Ghaleb-alnakhlani** [comment]
Any updates on this ?

**Comment by kimmeng168** [comment]
I also need that feature as well. Any update please
